exercises/phone-number/.meta/src/reference/kotlin/PhoneNumber.kt

```diff
@@ -8,14 +8,15 @@ data class PhoneNumber(private val rawNumber: String) {
     val number: String?
 
     init {
-        val containsInvalidChar = invalidChars.containsMatchIn(rawNumber)
-
-        if (containsInvalidChar) {
-            number = null
-        } else {
-            val digits = rawNumber.replace(Regex("[^\\d]"), "")
-            number = digitsValidationRegex.matchEntire(digits)?.groupValues?.last()
+        if (invalidChars.containsMatchIn(rawNumber)) {
```

Comment:
Maybe `require()` would be more fitting here?

Reply:
Indeed!